source/math/FloatSize3D.ooc

```diff
@@ -21,59 +21,59 @@ import IntSize3D
 use ooc-base
 
 FloatSize3D: cover {
-	width, height, depth: Float
-	volume ::= this width * this height * this depth
+	x, y, z: Float
+	volume ::= this x * this y * this z
 	length ::= this norm
-	empty ::= this width == 0 || this height == 0 || this depth == 0
-	norm ::= (this width squared() + this height squared() + this depth squared()) sqrt()
-	azimuth ::= this height atan2(this width)
+	empty ::= this x == 0 || this y == 0 || this z == 0
```

Comment:
I know that it was like that before, but how does it work in ooc, is the Int converted to Float or Float to Int for comparison ? Shouldn't we use a kind of `floatIsZero(this x)` anyway ? Not this PR just asking.